content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -683,6 +687,62 @@ More configuration options can be found on the [kong-plugin-ddtrace][3] plugin d
 [3]: https://github.com/DataDog/kong-plugin-ddtrace#configuration
 
 {{% /tab %}}
+
+{{% tab "Apache HTTPd" %}}
```

Comment:
```suggestion
{{% tab "Apache HTTPd / IBM HTTP Server (IHS)" %}}
```

Reply:
That will generate a gigantic tab. I suggestion to mention IHS in compatibility section instead.

Reply:
That works for me